README.md

```diff
@@ -161,9 +161,10 @@ Consider file test.proto, containing
 	  required string label = 1;
 	  optional int32 type = 2 [default=77];
 	  repeated int64 reps = 3;
-	  optional group OptionalGroup = 4 {
-	    required string RequiredField = 5;
+	  message OptionalNest {
```

Comment:
This is not valid proto syntax. You can't inline a message like this.

I would just remove the entire reference to groups.

Reply:
You're right, made a dumb type there. Will just remove the entire reference.